interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -14,7 +14,16 @@ angular.module('adage.analyze.sampleBin', [
 
 .factory('SignatureSet', ['$resource', 'ApiBasePath',
```

Comment:
That is a good idea, but out of scope for this PR. I'd like to keep this focused on issue #152.